src/cryptography/hazmat/bindings/commoncrypto/binding.py

```diff
@@ -4,54 +4,19 @@
 
 from __future__ import absolute_import, division, print_function
 
-import threading
-
-from cryptography.hazmat.bindings.utils import (
-    build_ffi_for_binding, load_library_for_binding,
-)
+try:
+    from cryptography._cryptography_commoncrypto import ffi, lib
+except ImportError:
```

Comment:
Do we need this? Can we just not support importing ``cryptography.hazmat.bindings.commoncrypto`` if we're not on a platform that has CommonCrypto?